### PR TITLE
Feature/commons rdbms

### PR DESCRIPTION
**Pull request by wkhessairi**

### CODE REVIEW

File: webapp/juzu-portlet/src/main/java/org/exoplatform/social/portlet/userNotification/templates/index.gtmpl [hunk 1]
```diff
@@ -78,7 +78,7 @@
 						int groupActive = 0;
 						for (def plugin : pluginInfos) {
 							String pluginId = plugin.getType();
-							if(plugin.getAllChannelActive().isEmpty()) {
+							if(plugin.getAllChannelActive().isEmpty() || (plugin.getAllChannelActive().size() == 1 && plugin.getAllChannelActive().get(0).equals("null"))) {
```
Comment (thomasdelhomenie) [inline webapp/juzu-portlet/src/main/java/org/exoplatform/social/portlet/userNotification/templates/index.gtmpl]: Looks like a workaround. In which case do we have 1 channel which equals to "null" ?

Comment (wkhessairi) [inline webapp/juzu-portlet/src/main/java/org/exoplatform/social/portlet/userNotification/templates/index.gtmpl]: In fact when we disable all notifications for a specific plugin, we get allChannelActive containing one element which is null. I don't remember why exactly we got this, but perhaps you are right we should focus at its origin.

Comment (thomasdelhomenie) [inline webapp/juzu-portlet/src/main/java/org/exoplatform/social/portlet/userNotification/templates/index.gtmpl]: Yes we should focus on its origin instead, at least to understand why and to know if we can fix it.

File: component/core/src/main/java/org/exoplatform/social/core/storage/cache/CachedActivityStorage.java [hunk 2]
```diff
@@ -231,7 +233,8 @@ public ActivityData execute() {
                 return ActivityData.NULL;
               }
             } catch (Exception e) {
-              throw new RuntimeException(e);
+              LOG.error("failed to get activity with id: " + activityId, e);
```
Comment (thomasdelhomenie) [inline component/core/src/main/java/org/exoplatform/social/core/storage/cache/CachedActivityStorage.java]: To be removed.

**Pull request merged**